pkg/platform/kube/platform.go

```diff
@@ -889,6 +897,10 @@ func (p *Platform) ResolveDefaultNamespace(defaultNamespace string) string {
 	}
 
 	if defaultNamespace == "" {
+		if p.DefaultNamespace != "" {
```

Comment:
same, if they would be resolved before, at this point you'll be looking at one variable